README.md

```diff
@@ -23,6 +23,8 @@ Create CRDs
 
     oc create -f deploy/crds/neutron_v1_neutronovsagent_crd.yaml
     oc create -f deploy/crds/neutron_v1_neutronsriovagent_crd.yaml
+    oc create -f deploy/crds/neutron.openstack.org_ovsnodeosps_crd.yaml
```

Comment:
you will need to create one or the other, right? perhaps worth to split it by ovs and ovn

Reply:
We will remove ovs so will update README then